packages/11ty/.eleventy.js

```diff
@@ -171,7 +171,7 @@ module.exports = function(eleventyConfig) {
    * @property {Boolean} useTransform - Use WebC transform to process all HTML output
    */
   eleventyConfig.addPlugin(pluginWebc, {
-    components: '_includes/components/**/*.webc',
+    components: ['_includes/components/**/*.webc', '_components/**/*.webc'],
```

Comment:
Listing the default, which is `'_components/**/*.webc'`, first.
```suggestion
    components: ['_components/**/*.webc', '_includes/components/**/*.webc'],
```